scripts/run-benchmarks.py

```diff
@@ -0,0 +1,185 @@
+import subprocess
+import json
+import os
+import re
+
+import click
+import cpuinfo
+
+BASE_REPO_URL = 'https://github.com/lumol-org/lumol.git'
+
+INDIVIDUAL_BENCH_TEMPLATE = """
+<details><summary>%s %s</summary>
+  <p>\n\n
+\n\n
+```bash\n
+%s\n\n
+```\n\n
+
+</p></details>
+"""
+
+COMPARISON_TEMPLATE = """
+<details><summary>%s %s</summary>
```

Comment:
I prefer if we only use one version throughout the code, to make it consistent. The `format` and bracketed version is the 'modern' version, and main version in Python 3, that's why I am pushing for it 😃